spec/serializers/spree/v2/storefront/vendor_image_serializer_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe Spree::V2::Storefront::VendorImageSerializer do
+  let!(:vendor_image) { create(:vendor_image) }
+  let!(:vendor) { create(:active_vendor, name: 'vendor', image: vendor_image) }
+
+  subject { described_class.new(vendor_image) }
+
+  it { expect(subject.serializable_hash).to be_kind_of(Hash) }
+
+  it do
+    expect(subject.serializable_hash).to eq(
+      {
+        data: {
+          id: vendor_image.id.to_s,
+          type: :vendor_image,
+          :attributes => {
+            :viewable_id => vendor.id,
+            :viewable_type => vendor.class.name,
+            :styles => [
+              { :height => "48", :url => "/rails/active_storage/representations/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBCZz09IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--8a45a5071ebace71ba86fe762e3b7a06ebcfc6c0/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdCem9MWm05eWJXRjBTU0lJYW5CbkJqb0dSVlE2QzNKbGMybDZaVWtpQ3pRNGVEUTRQZ1k3QmxRPSIsImV4cCI6bnVsbCwicHVyIjoidmFyaWF0aW9uIn19--ab6fb539515f81f7d5e69715bc961d15d9d43f48/thinking-cat.jpg", :width => "48" },
```

Comment:
also, I didn't see image related serializer specs in other spree projects,

please let me know if there is room for improvement in this spec as well :)